lib/Qandidate/Toggle/Serializer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the qandidate/toggle package.
+ *
+ * (c) Qandidate.com <opensource@qandidate.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Qandidate\Toggle;
+
+/**
+ * Representation of a serializer
+ *
+ * This class provides common ground for concrete serializer implementations
+ *
+ * @TODO: serialize() and deserialize() should be part of abstract interface
+ */
+abstract class Serializer
```

Comment:
If you have a concrete proposal for that can you put it in a separate PR? I'm happy to merge the phpdoc fixes + the `EqualsTo` bug fix, but I'd like to further discuss the new abstract class. :)